README.rst

```diff
@@ -1,38 +1,31 @@
 ZMON
 ====
 
-ZMON is Zalando's open source platform monitoring tool. It is in production use since early 2014 and supports our large number of teams in observing their services and metrics on various layers from CPU load to Team KPIs.
+ZMON is Zalando's open-source platform monitoring tool, used in production since early 2014. It supports our many engineering teams in observing their services and metrics on various layers, from CPU load to team KPIs.
 
 .. image:: https://readthedocs.org/projects/zmon/badge/?version=latest
    :target: https://readthedocs.org/projects/zmon/?badge=latest
    :alt: Documentation Status
 
-Take a look at slides from our recent talk at DevOps Ireland Meetup:
+Take a look at the slides from our [recent talk at the DevOps Ireland meetup](https://tech.zalando.com/blog/zmon-zalandos-open-source-monitoring-tool-slides/) for background information on ZMON.
 
-  https://tech.zalando.com/blog/zmon-zalandos-open-source-monitoring-tool-slides/
-
-A introduction on how it started:
-
-  https://tech.zalando.com/blog/monitoring-the-zalando-platform/
-
-Documentation:
-
-  http://zmon.readthedocs.org/en/latest/intro.html
+[Here's an introduction](https://tech.zalando.com/blog/monitoring-the-zalando-platform/) on how we started the project.
 
+[Here's more detailed documentation](http://zmon.readthedocs.org/en/latest/intro.html) than you'll find here.
 
 Features / Working:
- * Define checks as data sources executed on self defined entities
- * Define alerts on checks and entities as it fits your team with thresholds
- * Define custom dashboards with widgets and alert filters base on teams and tags
- * Check command and Alert condition are arbitrary Python expressions, giving you a lot of power
- * Store all results, incl. flattened dicts in time series in KairosDB
- * Internal charting and integrated Grafana for dash boards
- * Entity service to push entities, e.g. hosts, databases, ... we push ec2, elbs, ... via zmon-aws-agent
- * Define checks via yaml files and push using zmon-cli or via discovery from git sources
- * Use trial run in the UI to develop your checks/alerts with quick feedback
- * Trigger instant evaluation from UI for checks on longer intervals to rerun commands
-
-Start Demo using Vagrant
+ * Defines checks as data sources executed on self-defined entities
```

Comment:
OK. I was trying to achieve grammar-related consistency among the bullet points (ie make them all start with verbs).